src/SonarQube.Client/Models/ServerSentEvents/SSEStreamReader.cs

```diff
@@ -52,8 +52,6 @@ internal class SSEStreamReader : ISSEStreamReader
         private readonly IDictionary<string, Type> eventTypeToDataTypeMap = new Dictionary<string, Type>
```

Comment:
Maybe we can run Resharper's code cleanup + reformat on this class.
It has some unused usings

Reply:
Sorry, I will skip this. Need to move on.